src/protocol-curr/xdr/Stellar-overlay.x

```diff
@@ -27,6 +27,12 @@ struct SendMore
     uint32 numMessages;
 };
 
+struct SendMoreExtended
+{
+    uint32 numMessages;
+    uint32 numBytes;
```

Comment:
for overlay, we normally introduce new messages instead of extending old ones, since older overlay versions become obsolete and this approach makes it easy to deprecate old behavior (note that this is different from protocol, where we need to support old behavior forever)